examples/examples_dotnet_test.go

```diff
@@ -26,7 +26,10 @@ import (
 func TestNginxCs(t *testing.T) {
 	test := getCsharpBaseOptions(t).
 		With(integration.ProgramTestOptions{
-			Dir: path.Join(getCwd(t), "nginx-cs"),
+			Dir:              path.Join(getCwd(t), "nginx-cs"),
+			Quick:            true,
+			SkipRefresh:      true,
```

Comment:
@guineveresaenger the tests look like they may still refreshing on each run.. any thoughts?